app/code/Magento/QuoteGraphQl/Model/Resolver/SetShippingMethodsOnCart.php

```diff
@@ -69,13 +69,13 @@ public function resolve(Field $field, $context, ResolveInfo $info, array $value
 
         $shippingMethod = reset($shippingMethods); // This point can be extended for multishipping
 
-        if (!$shippingMethod['cart_address_id']) {
+        if (!isset($shippingMethod['cart_address_id'])) {
```

Comment:
Also, need to check that value is not empty

Reply:
add a condition to check empty value. Merge conflict was resolved